### PR TITLE
doc(example): pytorch torchhub yolov5 example

### PR DESCRIPTION
**Pull request by bojiang**

### CODE REVIEW

File: examples/pytorch_yolov5_torchhub/.jukit/.jukit_info.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"import_complete": 1, "terminal": "nvimterm"}
```
Comment (aarnphm) [inline examples/pytorch_yolov5_torchhub/.jukit/.jukit_info.json:1]: Is this file needed to check in?

File: examples/pytorch_yolov5_torchhub/tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    subprocess.run(cmd, shell=True, check=True)
+
+
+@pytest.fixture(scope="session")
+def host() -> t.Generator[str, None, None]:
+    import bentoml
+
+    bentoml.build("service:svc")
+
+    with run_api_server(
```
Comment (aarnphm) [inline examples/pytorch_yolov5_torchhub/tests/conftest.py:27]: I don't think we have this api any more

File: examples/pytorch_yolov5_torchhub/bentofile.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+service: "service:svc"
+description: "file: ./README.md"
+labels:
+  owner: bentoml-team
+  stage: demo
+include:
+- "*.py"
+- "/home/agent/.cache/torch/hub/ultralytics_yolov5_master/models/common.py"
```
Comment (ssheng) [inline examples/pytorch_yolov5_torchhub/bentofile.yaml:8]: Where do users get this file?

File: examples/pytorch_yolov5_torchhub/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Usage: pip install -r requirements.txt
+
+# Bento requirements
+pydantic
+
+
```
Comment (aarnphm) [inline examples/pytorch_yolov5_torchhub/requirements.txt:6]:
```suggestion
```

**Comment by bojiang** [comment]
close because of duplicate example
